appveyor.yml

```diff
@@ -5,15 +5,17 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version x64
   - node --version
-  - yarn --version
-  - yarn install
+  - curl -fsSL -o yarn.js https://github.com/yarnpkg/yarn/releases/download/v0.27.5/yarn-0.27.5.js
```

Comment:
Wouldn't `npm install --global yarn@0.27.5` be nicer here than `node ./yarn.js`?